README.md

```diff
@@ -34,6 +34,14 @@ the source code, run the tests, and contribute your changes to the project.
 
 :sparkles: Thanks to all our [contributors]! :sparkles:
 
+## Security
+
+If you think you found a vulnerability or a critical bug in the Arduino CLI,
+please read our [security policy] and report the bug to our 
```

Comment:
I am a little concerned that, despite all the context, the term "critical bug" might lead people to report issues that aren't related to security to the security email address. What do you think about this?
```suggestion
If you think you found a vulnerability or other security-related bug in the
Arduino CLI, please read our [security policy] and report the bug to our
```

Reply:
Great! :+1: